src/components/combo/combo.ts

```diff
@@ -325,6 +326,19 @@ export default class IgcComboComponent<T extends object>
     return html`${this.groupKey && item[this.groupKey]}`;
   };
 
+  /**
+   * Sets the component's positioning strategy.
+   */
+  @property({ attribute: 'position-strategy' })
+  public positionStrategy: 'absolute' | 'fixed' = 'fixed';
+
+  /**
+   * Whether the dropdown's width should be the same as the target's one.
+   * True by default.
+   */
+  @property({ type: Boolean, attribute: false })
+  public sameWidth = true;
```

Comment:
```suggestion
  /**
   * Whether the dropdown's width should be the same as the target's one.
   * True by default.
   */
  @property({ type: Boolean, attribute: 'same-width' })
  public sameWidth = false;
```
This should match https://github.com/IgniteUI/igniteui-webcomponents/blob/0a24a42356476bff4dc16df8863792a523905bc9/src/components/select/select.ts#L188-L193

Reply:
👀 Oh we have `sameWidth` true by default on the select? Well that's awkward.
I was going to comment the position strategy should match as well.